index.js

```diff
@@ -15,11 +16,20 @@ var nameMap = {
 };
 
 module.exports = function (release) {
-	var version = /\d+\.\d+/.exec(release || os.release());
+	var verRe = /\d+\.\d+/;
+	var version;
 
-	if (!version) {
+	if (release || semver.gte(process.version, '3.1.0')) {
+		version = verRe.exec(release || os.release());
+	} else {
+		// Workaround for Windows 10 on node < 3.1.0
+		var execSync = require('child_process').execSync;
+		version = verRe.exec(String(execSync('ver.exe', {timeout: 1e5})));
```

Comment:
Hmm, I get that with the native method too:

```
> require('child_process').execFileSync('ver').toString()
TypeError: invalid data
    at WriteStream.Socket.write (net.js:633:11)
    at Object.execFileSync (child_process.js:1364:20)
    at repl:1:26
    at REPLServer.defaultEval (repl.js:132:27)
    at bound (domain.js:254:14)
    at REPLServer.runBound [as eval] (domain.js:267:12)
    at REPLServer.<anonymous> (repl.js:279:12)
    at REPLServer.emit (events.js:107:17)
    at REPLServer.Interface._onLine (readline.js:214:10)
    at REPLServer.Interface._line (readline.js:553:8)
```

I guess we can just do the workaround on 0.12.0 and higher.